docs/01-Using-Fleet/04-Adding-hosts.md

```diff
@@ -25,9 +26,22 @@ fleetctl package --type pkg --fleet-url=[YOUR FLEET URL] --enroll-secret=[YOUR E
 
 When you install the generated osquery installer on a host, this host will be automatically enrolled in the specified Fleet instance.
 
+## Signing installers
+
+  >**Note:** Windows signing is not yet supported within `fleetctl` (working on that this week), but the resulting MSI could be signed after building.
+
+`fleetctl` provides suppport for signing and notarizing macOS osquery packages via the `package` command. To do
+this, modify the command to create an osquery installer to look with the `--sign-identity` and
+`--notarize` flags like the example below:
```

Comment:
```suggestion
The `fleetctl package` command provides suppport for signing and notarizing macOS osquery packages via the 
`--sign-identity` and `--notarize` flags. 

Check out the example below:

Reply:
I propose combining the 2 sentences into 1 here. This way, the documentation is as brief as possible while still delivering the user clear instructions.